src/applications/vaos/tests/e2e/community-care.cypress.spec.js

```diff
@@ -159,6 +160,66 @@ describe('Create new community care appointment', () => {
     // Click continue button
     cy.get('.usa-button').click();
 
+    // Check form requestBody is as expected
+    cy.wait('@appointmentRequests').should(xhr => {
+      let date = moment().add(5, 'days');
+
+      if (date.weekday() === 0) {
```

Comment:
Might be good to put a comment in here explaining why we're doing this extra data logic.